packages/polyfills/src/metro.ts

```diff
@@ -0,0 +1,13 @@
+import type { MetroConfig } from "expo/metro-config";
+
+type Writeable<T> = { -readonly [P in keyof T]: Writeable<T[P]> };
+
+export function setupWakuMetroConfig(config: Writeable<MetroConfig>): void {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const path = require("path");
```

Comment:
any reason we aren't doing es6 imports?

Reply:
no in particular, es6 is longer so I usually prefer `require` style
changed here to see if it would work with `metro`